standards/qa-process.md

```diff
@@ -54,14 +56,14 @@
 1. Desired browsers and devices to be tested on are identified.
 2. Team is assigned to the project repo on Github in order to post issues and coordinate with developers. (This will be Write access)
 3. If applicable, review for each user types abilities and limitations.
-4. Credentials are given to the team.
+4. Credentials are given to the team via **Zoho Vault**. (BWTC-QA Chamber)
 5. If QA is billable to the client, such as for T&M projects, PM will add the QA team to the Tick project.
 
 # Testing Process
 ## For QA:
 
-1. QA will set up a project board for this sprint by copying from the example board on github (link).
-2. The milestone and labels relevant for this sprint will be added to ensure that items are displayed and sorted correctly on the project board.
+1. QA will set up a project board for this sprint by copying from the [example board](https://github.com/Shift3/qa-team/projects/9) on github.
```

Comment:
Disregard the CircleCI issue here. We just need to add https://github.com/Shift3/standards-and-practices/blob/main/.circleci/config.yml#L10 to the allowlist in a separate PR.